src/components/Conversion/ConversionSummary.vue

```diff
@@ -20,7 +20,7 @@
           </v-col>
 
           <v-col cols="12" sm="8">
-            <span class="info-text">{{ naicsSummary }}</span>
+            <span class="info-text">{{naicsSummary || '(Not entered)'}}</span>
```

Comment:
It should be the template's job to display specific text, not the script. (Except when the logic is overly complicated for the template.)